tests/lib/Persistence/Legacy/Content/Type/ContentTypeHandlerTest.php

```diff
@@ -887,44 +923,39 @@ public function testRemoveFieldDefinition()
             );
 
         $handler = $this->getHandler();
-        $res = $handler->removeFieldDefinition(23, 1, 42);
+        $res = $handler->removeFieldDefinition(23, 1, 42, 'ezstring');
 
         $this->assertTrue($res);
     }
 
     public function testUpdateFieldDefinition()
     {
+        $fieldDef = new FieldDefinition();
+
         $mapperMock = $this->getMapperMock(
             ['toStorageFieldDefinition']
         );
         $mapperMock->expects($this->once())
             ->method('toStorageFieldDefinition')
-            ->with(
-                $this->isInstanceOf(
-                    FieldDefinition::class
-                ),
-                $this->isInstanceOf(
-                    StorageFieldDefinition::class
-                )
-            );
+            ->with($fieldDef, $this->isInstanceOf(StorageFieldDefinition::class));
```

Comment:
If you'd like to check if it's the same object instance:

```suggestion
            ->with($this->identicalTo($fieldDef), $this->isInstanceOf(StorageFieldDefinition::class));
```

Otherwise it's loose equality check.